components/user/UserEddiehubData.js

```diff
@@ -0,0 +1,29 @@
+import UserEddiehubDataItem from "./UserEddiehubDataItem";
+
+export default function UserEddiehubData({ data }) {
+  const items = [
+    {
+      amount: data.issues,
+      label: "Issues",
+    },
+    {
+      amount: data.issueComment,
+      label: "Issue Comments",
+    },
+    {
+      amount: data.pullRequest,
+      label: "Pull Requests",
+    },
+    {
+      amount: data.pullRequestReview,
+      label: "Pull Request Reviews",
+    },
+  ];
+  return (
+    <div className="grid grid-cols-4 gap-8 mt-6">
```

Comment:
This needs improvements for mobile breakpoints

Reply:
Don't forget we have paid for TailwindUI components, so if you need anything from there let me know and I can get the component's code - that way the responsiveness is already done

Reply:
I used a tailwind component for displaying the items. This line is just for the layout and displaying them next to each other, so I think it's fine 👍